LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+BSD 3-clause license
```

Comment:
The license should be the apache 2 license file that already exists in the repo.

Reply:
This was copied over from forge, so we should keep their license and header.

Reply:
Ah, got it